src/platform/testing/e2e/form-tester/form-filler.js

```diff
@@ -34,7 +34,7 @@ const getElementSelector = (field, fieldData) => {
       typeof fieldData === 'boolean' ? (fieldData ? 'Y' : 'N') : fieldData
     }"]`,
     // Date has two or three elements, but should always have a year
-    // Return a valid selector so it doesn't get skippeG
```

Comment:
More cleanup. Has nothing whatsoever to do with this PR, but I didn't want to just leave it.